app/frontend/src/components/forms/SubmissionsTable.vue

```diff
@@ -140,6 +176,12 @@ export default {
       }
     },
   },
+
+  created(){
+    // Get the permissions for this form
+    this.getFormPermissionsForUser(this.formId);
+  },
+
```

Comment:
Why do we need to do getFormPermissionsForUser in the created lifecycle? It is sufficient enough to do this as part of the mounted lifecycle.

Reply:
i forget exactly why i was doing this on the created() event. I've put it back in mounted() and it seems to be working